zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -108,6 +108,7 @@
 
     @Getter
     @Inject
+    @Any
```

Comment:
Still not sure about the `@Any` in these injection points.
I thought it used to inject instances of `Instace<>` and `Event<>`